src/ImportExport.ts

```diff
@@ -475,7 +478,14 @@ export const promocodes = async (input: string | null) => {
 
         const ascensionSpeed = calculateAscensionAcceleration()
         const perSecond = 1/(24 * 3600 * 365) * baseMultiplier * productContents(valueMultipliers) * ascensionSpeed
-        return Alert(`You will gain an octeract (when they come out) every ${format(1 / perSecond, 2, true)} seconds, assuming you have them unlocked!`)
+        const perSecondText = perSecond >= 1 ? `${format(perSecond, 2, true)} per second` : `every ${format(1 / perSecond, 2, true)} seconds`
+        return Alert(`You will gain an octeract (when they come out) ${perSecondText}, assuming you have them unlocked!`)
+    } else if (input === 'date') {
```

Comment:
is this needed?

Reply:
Before Cx20 was nerfed, I've been able to get less than 0.1 with normal data.
Therefore, I think it may progress to 0 in future versions.
Ascension Speed is too powerful and I think it should be reduced by exponentiation.
If it's still not needed, I'll erase the code I pointed out.

Reply:
I got 0.72 seconds with spoiler at 10/14/14/14/14/14/14/14 of main data sing 33 in v2.9.8.
Upload data if necessary.
Should the date code be erased? I think Synergism needs a watch.
The following ImportExport.ts upload contains more fixes.

Reply:
I don't think adding this as a promocode is the solution

Reply:
ok, deleted